tests/functionaltests/Tests/CoreAnimationTests/CALayerAppearanceTests.mm

```diff
@@ -104,7 +106,7 @@ - (void)viewDidLoad {
     });
 
     [condition lock];
-    ASSERT_TRUE_MSG([condition waitUntilDate:[NSDate dateWithTimeIntervalSinceNow:c_testTimeoutInSec]],
+    ASSERT_TRUE_MSG(signaled || [condition waitUntilDate:[NSDate dateWithTimeIntervalSinceNow:c_testTimeoutInSec]],
```

Comment:
Ah, right.  I think Brian already had to go fix up a bunch of other tests regarding this same issue.  Can we use a Win32 or C++ construct for this?  That's a lot of code for what should be a simple wait.

---
In reply to: [94687381](https://github.com/Microsoft/WinObjC/pull/1627#discussion_r94687381) [](ancestors = 94687381,94669086)